### PR TITLE
Vrouter certificate is now imported in test phase

### PR DESCRIPTION
**Pull request by jacekchalupka**
vRouter is no longer signed with codilime cert, but with one that is
generated by Visual Studio and exported to vRouter.cer file.

### CODE REVIEW

File: CIScripts/Build.ps1 [hunk 1]
```diff
@@ -42,26 +42,16 @@ foreach ($Directory in $Directories) {
...
-        -LogsPath $LogsDir
+        -OutputPath $CnmPluginOutputDir
 
     Invoke-ExtensionBuild -ThirdPartyCache $Env:THIRD_PARTY_CACHE_PATH `
-        -SigntoolPath $Env:SIGNTOOL_PATH `
-        -CertPath $Env:CERT_PATH `
-        -CertPasswordFilePath $Env:CERT_PASSWORD_FILE_PATH `
         -BuildMode $SconsBuildMode `
         -OutputPath $vRouterOutputDir `
         -LogsPath $LogsDir
```
Comment (michal-clapinski) [inline CIScripts/Build.ps1]: Are those even useful anymore? You could remove those too :D

File: Test/Utils/ComputeNode/Installation.ps1 [hunk 2]
```diff
@@ -62,6 +62,34 @@ function Uninstall-Agent {
...
 }
 
+function Import-ExtensionCertificate {
+    Param ([Parameter(Mandatory = $true)] [Testbed] $Testbed)
+
+    Write-Log 'Importing vRouter certificate to certstore'
+    Invoke-Command -Session $Testbed.GetSession() {
+        $Cert = Import-Certificate -CertStoreLocation Cert:\LocalMachine\Root\ 'C:\Artifacts\vrouter\vRouter.cer'
+        try {
+            Import-Certificate -CertStoreLocation Cert:\LocalMachine\TrustedPublisher\ 'C:\Artifacts\vrouter\vRouter.cer'
```
Comment (michal-clapinski) [inline Test/Utils/ComputeNode/Installation.ps1]: why is one in `try` block but the other one is not? please add comment to the code about this

Comment (jacekchalupka) [reply Test/Utils/ComputeNode/Installation.ps1]: Isn't the catch block explaining why?

File: CIScripts/Build.ps1 (hunk 1 above)
Comment (jacekchalupka) [reply CIScripts/Build.ps1]: You mean logs?

Comment (michal-clapinski) [reply CIScripts/Build.ps1]: Yeah. I went to the logs folder and couldn't find `vrouter_build.log` for example. And even if it's there, it's rather useless, right?

File: Test/Utils/ComputeNode/Installation.ps1 (hunk 2 above)
Comment (jacekchalupka) [reply Test/Utils/ComputeNode/Installation.ps1]: Yes, that is the reason. Importing twice will not break anything, but the idea behind our tests is: cleanup after yourself if it is possible, leave the place as you found it. That's why all steps that we run during initialization should be atomic in a sense, that they either success or fail, but leave environment untouched. If the function is not working like this, it has CleanupStack as a parameter, and after every step it adds cleanup step.

File: CIScripts/Build.ps1 (hunk 1 above)
Comment (jacekchalupka) [reply CIScripts/Build.ps1]: I have no idea why it is here, and I believe I would have to look through code what happens with logs, I don't feel like deleting them without it :) But I will gladly do it tomorrow :)

Comment (michal-clapinski) [reply CIScripts/Build.ps1]: I'll do it. Already working on it while writing proper test runner

**Comment by jacekchalupka** [comment]
recheck no bug

**Pull request merged**